modules/recurlyjs/recurlyjs.module

```diff
@@ -121,3 +122,17 @@ function recurlyjs_settings_form_submit(array &$form, FormStateInterface $form_s
     ->set('recurlyjs_accept_paypal', $form_state->getValue('recurlyjs_accept_paypal'))
     ->save();
 }
+
+/**
+ * Implements hook_entity_type_alter().
+ */
+function recurlyjs_entity_type_alter(array &$entity_types) {
+  $recurly_entity_type = \Drupal::config('recurly.settings')->get('recurly_entity_type') ?: 'user';
```

Comment:
@adamzimmermann I agree with you. I brought this up with @markdorison. I believe the plan is address this at a later date.